src/ui/flutter_app/lib/widgets/game_page.dart

```diff
@@ -514,7 +519,7 @@ class _GamePageState extends State<GamePage>
     return ret;
   }
 
-  engineToGo(bool isMoveNow) async {
+  Future<void> engineToGo(bool isMoveNow) async {
```

Comment:
No suspicion has been found inside this function for the time being. Did the problem occur after the refactoring? Did the problem exist before there was refactoring?